sklearn/model_selection/_split.py

```diff
@@ -2126,6 +2126,8 @@ def train_test_split(*arrays,
     -------
     splitting : list, length=2 * len(arrays)
         List containing train-test split of inputs.
+        The order of the returned list is:
+        X_train, X_test, y_train, y_test.
```

Comment:
This would only apply when `train_test_split` is used with two arrays. But `train_test_split` can be used with more than two arrays (or one array)

```python
import numpy as np

from sklearn.datasets import make_regression
from sklearn.model_selection import train_test_split

X, y = make_regression()
sample_weights = np.random.rand(y.shape[0])

X_train, X_test, y_train, y_test, sw_train, sw_test = \
    train_test_split(X, y, sample_weights)
```

If we want to include this inline, I suggest adding the input as well:

```
        List containing train-test split of inputs. When passed two arrays
        the order of the splitting is::

            X_train, X_test, y_train, y_test = train_test_split(X, y)
```

Reply:
@thomasjpfan, thanks for your suggestions, which makes me understand this function further. Now I don't think the adding doc about the returned list is not necessary. We can read it from the examples below. Thanks.